common/src/java/org/apache/hadoop/hive/conf/HiveConf.java

```diff
@@ -779,6 +779,13 @@ private static void populateLlapDaemonVarsSet(Set<String> llapDaemonVarsSetLocal
     METASTORE_TRANSACTIONAL_EVENT_LISTENERS("hive.metastore.transactional.event.listeners", "",
         "A comma separated list of Java classes that implement the org.apache.hadoop.hive.metastore.MetaStoreEventListener" +
             " interface. Both the metastore event and corresponding listener method will be invoked in the same JDO transaction."),
+    NOTIFICATION_SEQUENCE_LOCK_MAX_RETRIES("hive.notification.sequence.lock.max.retries", 5,
```

Comment:
Start with uppercase and change 'retires' to 'retries'. 

Can I suggest the following message?
- Number of retries required to acquire a lock when getting the next notification sequential ID for entries in the NOTIFICATION_LOG table.

Reply:
done